src/loaders/compute_tools/eggnog/eggnog.py

```diff
@@ -22,7 +21,7 @@ def _run_eggnog_single(
         threads_per_tool_run: int,
         debug: bool) -> None:
 
-    metadata_file = output_dir / EGGNOG_METADATA
+    metadata_file = output_dir / TOOL_METADATA
     if metadata_file.exists():
         print(f"Skipping {source_file} as it has already been processed.")
         return
```

Comment:
When would this case occur?

Reply:
during a rerun - in the event of node timeout or just want to retry the failure ones. But never used before.

Reply:
Any so we're just replacing the old metadata file with the new one, which means a rerun on the old metadata file would rerun everything?

Reply:
yea. But I am not sure why we want to do that. The current finished collections are all in good place. If we want to rerun any collection means we mean a full rerun I guess. I am not aware of any failures in any of current collections.

Reply:
:+1: